set.mm

```diff
@@ -442237,9 +442237,9 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "digit" as 'digit';
  althtmldef "digit" as 'digit';
  latexdef "digit" as "\mathrm{digit}";
-htmldef "NaryF" as ' NaryF ';
- althtmldef "NaryF" as ' NaryF ';
- latexdef "NaryF" as "\mathrm{NaryF}";
+htmldef "-aryF" as ' -aryF ';
+ althtmldef "-aryF" as ' -aryF ';
```

Comment:
I think we should remove the the space at the beginning (' -aryF '=> '-aryF '), so that we will see `( N-aryF X )` instead of `( N -aryF X `).

Reply:
> I think we should remove the the space at the beginning (' -aryF '=> '-aryF '), so that we will see `( N-aryF X )` instead of `( N -aryF X `).

PS: not in set.mm, but in the the HTML-representation